bin/bootstrax

```diff
@@ -556,14 +572,14 @@ def process_run(rd, send_heartbeats=True):
                 # Since chunks can be empty, and we don't want to crash,
                 # this has to be done with some care...
                 t_covered = timedelta(seconds=(
-                    max([x.get('last_endtime', 0)
+                    max([x.get('last_endtime', 0) 
```

Comment:
Here you're only adding trailing whitespaces. You can configure pycharm to autoremove them, or check in gitkraken when you make the commit. Of course trailing whitespace doesn't really matter, but this would make the git blame a bit harder to read.